src/rules/delta/extended/in.jl

```diff
@@ -0,0 +1,57 @@
+@rule DeltaFn{f}((:in, _), Marginalisation) (
+    m_out::Any,
+    m_ins::Nothing,
+    meta::DeltaExtended{T}
```

Comment:
Why is the meta a `DeltaExtended`, and not just `Extended` (or `Linearized` as Bart suggested)?